src/test/java/org/broadinstitute/hellbender/tools/spark/sv/SVIntervalTreeTest.java

```diff
@@ -0,0 +1,244 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Iterator;
+
+public class SVIntervalTreeTest extends BaseTest {
+    private static SVInterval[] intervals = {
+            new SVInterval(0, 100, 200),
+            new SVInterval(0, 150, 250),
+            new SVInterval(0, 200, 300),
+            new SVInterval(0, 250, 350),
+            new SVInterval(0, 300, 400),
+            new SVInterval(0, 350, 450),
+            new SVInterval(0, 400, 500),
+            new SVInterval(0, 450, 550),
+            new SVInterval(0, 500, 600),
+            new SVInterval(0, 550, 650),
+            new SVInterval(0, 600, 700),
+            new SVInterval(0, 650, 750),
+            new SVInterval(0, 700, 800),
+            new SVInterval(0, 750, 850),
+            new SVInterval(0, 800, 900),
+            new SVInterval(0, 850, 950),
+            new SVInterval(0, 900, 1000),
+            new SVInterval(1, 0, 100)
+    };
+
+    private static SVIntervalTree<Integer> initTree() {
+        final SVIntervalTree<Integer> tree = new SVIntervalTree<>();
+        final int[] loadOrder = {1, 5, 3, 15, 11, 7, 13, 9, 14, 4, 17, 12, 0, 8, 16, 2, 6, 10};
+        Assert.assertEquals(intervals.length, loadOrder.length);
+        for ( final int idx : loadOrder ) {
+            tree.put(intervals[idx], idx);
+        }
+        return tree;
+    }
+
+    @Test(groups = "spark")
+    public void sizeAndClearTest() {
+        final SVIntervalTree<Integer> testTree = initTree();
+        Assert.assertEquals(testTree.size(), intervals.length);
+        testTree.clear();
+        Assert.assertEquals(testTree.size(), 0);
+    }
+
+    @Test(groups = "spark")
```

Comment:
I haven't quite figured out our policy.  I saw some other code that had introduced an "sv" group, and thought that made sense.  But I guess I haven't learned to use "sv" consistently.
We'd like to group all our test methods in the future under group "sv"?  Is that right?
I'll change these to "sv".